PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -363,13 +386,16 @@ log_Msg "T2wFolder: $T2wFolder"
 log_Msg "AtlasSpaceFolder: $AtlasSpaceFolder"
 
 # Unpack List of Images
-T1wInputImages=`echo ${T1wInputImages} | sed 's/@/ /g'`
-T2wInputImages=`echo ${T2wInputImages} | sed 's/@/ /g'`
+T1wInputImages=`echo ${T1wInputImages} | sed 's/@/ /g' | sed -e  's/^[ \t]*//'`
+T2wInputImages=`echo ${T2wInputImages} | sed 's/@/ /g' | sed -e  's/^[ \t]*//'`
```

Comment:
This suggests that someone's setup was passing whitespace or an extra `@` on the front of the argument value.  Since the loop over images uses an unquoted variable, extra whitespace would get ignored anyway, so I'm not sure what the purpose of this is.